content/en/docs/api-reference/storage-system-installation/glusterfs-server.md

```diff
@@ -0,0 +1,510 @@
+   ```bash
+   ssh-keygen
+   ```
+
+2. Copy the key to all GlusterFS nodes.
+
+   ```bash
+   ssh-copy-id root@server1
+   ```
+
+   ```bash
+   ssh-copy-id root@server2
+   ```
+
+   ```bash
+   ssh-copy-id root@server3
+   ```
+
+3. Verify that you can access all server machines from `server1` through passwordless login.
+
+   ```bash
+   ssh root@server1
+   ```
+
+   ```bash
+   ssh root@server2
+   ```
+
+   ```bash
+   ssh root@server3
+   ```
+
+## Install GlusterFS on All Server Machines
+
+1. On `server1`, run the following command to install `software-properties-common`.
+
+   ```bash
+   apt-get install software-properties-common
+   ```
+
+2. Add the community GlusterFS PPA.
+
+   ```bash
+   add-apt-repository ppa:gluster/glusterfs-7
+   ```
+
+3. Make sure you are using the latest package.
+
+   ```bash
+   apt-get update
+   ```
+
+4. Install the GlusterFS server.
+
+   ```bash
+   apt-get install glusterfs-server -y
+   ```
+
+5. Make sure you run the above commands on `server2` and `server3` as well and verify the version on all machines.
+
+   ```text
+   glusterfs -V
+   ```
+
+{{< notice note >}}
+
+The above commands may be slightly different if you do no install GlusterFS on Ubuntu. For more information, see [the Gluster documentation](https://docs.gluster.org/en/latest/Install-Guide/Install/#installing-gluster).
+
+{{</ notice >}} 
+
+## Load Kernel Modules
+
+1. Run the following commands to load three necessary kernel modules on `server1`.
+
+   ```bash
+   echo dm_thin_pool | sudo tee -a /etc/modules
+   ```
+
+   ```bash
+   echo dm_snapshot | sudo tee -a /etc/modules
+   ```
+
+   ```bash
+   echo dm_mirror | sudo tee -a /etc/modules
+   ```
+
+2. Intall `thin-provisioning-tools`.
+
+   ```bash
+   apt-get -y install thin-provisioning-tools
+   ```
+
+3. Make sure you run the above commands on `server2` and `server3` as well.
+
+## Create a GlusterFS Cluster
+
+1. Run the following command on `server1` to add other nodes and create a cluster.
+
+   ```bash
+   gluster peer probe server2
+   ```
+
+   ```
+   gluster peer probe server3
+   ```
+
+2. Verify that all nodes in the cluster are connected successfully.
+
+   ```bash
+   gluster peer status
+   ```
+
+3. Expected output:
+
+   ```bash
+   Number of Peers: 2
+   
+   Hostname: server2
+   Uuid: e1192d6a-b65e-4ce8-804c-72d9425211a6
+   State: Peer in Cluster (Connected)
+   
+   Hostname: server3
+   Uuid: 9bd733e4-96d4-49d5-8958-6c947a2b4fa6
+   State: Peer in Cluster (Connected)
+   ```
+
+## Install Heketi
+
+As GlusterFS itself does not provide a way for API calls, you can install [Heketi](https://github.com/heketi/heketi) to manage the lifecycle of GlusterFS volumes with a RESTful API for Kubernetes calls. In this way, your Kubernetes cluster can dynamically provision GlusterFS volumes. Heketi v7.0.0 will be installed in this example. For more information about available Heketi versions, see its [Release Page](https://github.com/heketi/heketi/releases/).
```

Comment:
Should explicitly say which server should be operated with following content

Reply:
Changed.